src/main/scala/org/globalforestwatch/summarystats/gfwpro_dashboard/GfwProDashboardAnalysis.scala

```diff
@@ -45,13 +43,20 @@ object GfwProDashboardAnalysis extends SummaryAnalysis {
             spark: SparkSession,
             kwargs: Map[String, Any]): Unit = {
 
-    val enrichedRDD = intersectWithContextualLayer(featureRDD, kwargs, spark)
+    // only create this once for all analyses, analyze() can be slow
+    val spatialFeatureRDD = new SpatialRDD[GeoSparkGeometry]()
+    spatialFeatureRDD.rawSpatialRDD = featureRDD
+      .map(fromGeotrellisFeature[Geometry, FeatureId, GeoSparkGeometry])
+      .toJavaRDD()
+    spatialFeatureRDD.analyze()
```

Comment:
we will need to call `spatialFeatureRDD.cache()` here to cache it in memory, otherwise Spark will run this multiple times